Tests/BeeDatasetTests/BeePPCATests.swift

```diff
@@ -0,0 +1,94 @@
+import BeeDataset
+import SwiftFusion
+import TensorFlow
+import XCTest
+
+import PenguinStructures
+
+func calculatePPCAParams() -> (W: Tensor<Double>, mu: Tensor<Double>) {
```

Comment:
I am not loving the use of globals, and at the very least this function should be documented and it should be explained which globals are being used. I'd prefer it if you make this function take arguments and only use the globals in the call. I'd prefer it even more if you did not use globals at all :-)

Reply:
removed in favor of the generic version